src/demo-app/list/list-demo.ts

```diff
@@ -69,4 +69,8 @@ export class ListDemo {
     this.selectedOptions = values;
     this.modelChangeEventCount++;
   }
+
+  onButtonClick(msg: string) {
```

Comment:
Prefer to name methods for what they do rather than how they're called, e.g. `alertItem(...)`

(this doesn't really matter for demos, but I mention it just as an FYI)